opencti-platform/opencti-graphql/src/modules/attributes/internalObject-registrationAttributes.ts

```diff
@@ -78,6 +78,7 @@ const internalObjectsAttributes: { [k: string]: Array<AttributeDefinition> } = {
   ],
   [ENTITY_TYPE_USER]: [
     { name: 'user_email', type: 'string', mandatoryType: 'external', multiple: false, upsert: false },
+    { name: 'dashboard_id', type: 'string', mandatoryType: 'external', multiple: false, upsert: true },
```

Comment:
dashboard_id is not mandatory if we return null is some case
What do you think ?

Reply:
I do and I also do reckon I need more info on those options. Would you point me a resource on that matter, or take a little bit of time to explain me directly?

Reply:
I can explain to you directly, because we don't have clear documentation on this